packages/analytics/lib/index.js

```diff
@@ -102,20 +102,10 @@ class FirebaseAnalyticsModule extends FirebaseModule {
     return this.native.setAnalyticsCollectionEnabled(enabled);
   }
 
-  setCurrentScreen(screenName, screenClassOverride) {
-    if (!isString(screenName)) {
-      throw new Error(
-        "firebase.analytics().setCurrentScreen(*) 'screenName' expected a string value.",
-      );
-    }
-
-    if (!isUndefined(screenClassOverride) && !isString(screenClassOverride)) {
-      throw new Error(
-        "firebase.analytics().setCurrentScreen(_, *) 'screenClassOverride' expected a string value.",
-      );
-    }
-
-    return this.native.setCurrentScreen(screenName, screenClassOverride);
+  setCurrentScreen() {
```

Comment:
Can this after the warning just pass along the args to `logScreenView` method in the JS class, so it still continues to function?